src/lib/rest/httpRequestSend.cpp

```diff
@@ -517,8 +517,15 @@ int httpRequestSendWithCurl
   }
 
 
+  //
+  // FIXME: outgoingMsgSize is a signed int. Max size ~2GB. Might not be enough.
+  //        Might be a good idea to change it to 'unsigned long long'
+  //
+
+  //
   // Check if total outgoing message size is too big
-  if (outgoingMsgSize > MAX_DYN_MSG_SIZE)
+  //
+  if ((unsigned long long) outgoingMsgSize > outMsgMaxSize)
```

Comment:
This limit is defined for "response, forward, and notification". However, taking into account the place where it is used I'd say that only applies to "forward and notifications". That's fine, but let's take into account to document it properly.

Reply:
Yes, you're right.
The responses are taken care of using MHD, inside restReply().
Too many years ago ...

Reply:
To be clear: I'm not meaning modifying the implementation so outMsgMaxSize also controls responses sizes. What I mean in change documentation, CNR, etc. to match the implementation in the PR and tell just "forward and notification".

This PR is about making existing static limits dynamic not about adding new limits.

Reply:
Fixed in 636d28252 (in usage and in CNR. Hope that's all)